includes/ets_user_qa_function.php

```diff
@@ -196,380 +197,411 @@ public function ets_ask_qustion_tab() {
+									<td colspan="2"><p> <?php echo $value['answer']; ?></p></td> 
 								</tr> 
-							<?php 
+								<?php
 							} else {
-							?>
+								?>
 								<tr>
-									<td class="ets-question-title"><p><?php echo __('Answer:','product-questions-answers-for-woocommerce'); ?></p></td>
-									<td colspan="2" class="ets-no-answer" ><h6><p><i><?php echo __("Answer awaiting",'product-questions-answers-for-woocommerce');?>...</i></p></h6></td>	
+									<td class="ets-question-title"><p><?php echo __( 'Answer:', 'product-questions-answers-for-woocommerce' ); ?></p></td>
+									<td colspan="2" class="ets-no-answer" ><h6><p><i><?php echo __( 'Answer awaiting', 'product-questions-answers-for-woocommerce' ); ?>...</i></p></h6></td>	
 								</tr> 
 								<?php
 							}
 							$count++;
-							if($count > $productQaLength){
+							if ( $count > $productQaLength ) {
 								break;
-							}  
-							
-						} ?>
+							}
+						}
+						?>
 						</tbody>
 						</table>  
 					</div>
-						<?php					
+						<?php
 				}
-				 	?>  
+				?>
+					  
 				<button type="submit" id="ets-load-more" class="btn btn-success ets-qa-load-more" name="ets_load_more" value=""><?php echo $loadMoreButtonName; ?></button>
 				<div class="ets_pro_qa_length"><p hidden><?php echo $keyData; ?></p></div><div id="ets_product_qa_length"><p></p></div>
 	
 				<?php
 			}
 		} else {
-			//Show Question Answer Listing Type Table With OUt Load More
-			if(!empty($etsGetQuestion)){ 
+			// Show Question Answer Listing Type Table With OUt Load More
+			if ( ! empty( $etsGetQuestion ) ) {
 				?>
 				<div class="table-responsive my-table">
 				<table class="table table-striped"> 
 				<?php
-				foreach ($etsGetQuestion as $key => $value) {
-					?> 
+				foreach ( $etsGetQuestion as $key => $value ) {
+					?>
+					 
 					<tr class="ets-question-top">
-							<td class="ets-question-title"><p><?php echo __('Question','ets_q_n_a'); ?>:</p></td>
-							<td class="ets-question-description"><p><?php echo $value['question'];?></p></td> 
-							<td class="ets-cont-right"><h6 class="user-name"><?php echo $value['user_name'] . "<br>";    
-							echo ($value['date']);
-							?></h6></td>
+							<td class="ets-question-title"><p><?php echo __( 'Question', 'ets_q_n_a' ); ?>:</p></td>
+							<td class="ets-question-description"><p><?php echo $value['question']; ?></p></td> 
+							<td class="ets-cont-right"><h6 class="user-name">
+							<?php
+							echo $value['user_name'] . '<br>';
+							echo ( $value['date'] );
+							?>
+							</h6></td>
 					</tr>
 
-					<?php 
-					if(!empty($value['answer'])){?>
+					<?php
+					if ( ! empty( $value['answer'] ) ) {
+						?>
 						<tr>
-							<td class="ets-question-title"><p><?php echo __('Answer','ets_q_n_a'); ?>:</p></td>
-							<td colspan="2"><p> <?php echo $value['answer'];?></p></td> 
+							<td class="ets-question-title"><p><?php echo __( 'Answer', 'ets_q_n_a' ); ?>:</p></td>
```

Comment:
Please update text domain in translation func here.

Reply:
> Please update text domain in translation func here.

Already updated.
dont find any text domain : 'ets_q_n_a' in the plugin files